primitives/src/pcs/univariate_kzg/srs.rs

```diff
@@ -41,9 +41,9 @@ impl<E: Pairing> UnivariateUniversalParams<E> {
 
 /// `UnivariateProverParam` is used to generate a proof
 #[derive(CanonicalSerialize, CanonicalDeserialize, Clone, Debug, Eq, PartialEq, Default)]
-pub struct UnivariateProverParam<C: AffineRepr> {
+pub struct UnivariateProverParam<E: Pairing> {
     /// Config
-    pub powers_of_g: Vec<C>,
+    pub powers_of_g: Vec<E::G1Affine>,
```

Comment:
There's no technical reasons why we can't, it's just visual consistency. 
Imo it would be more obvious to associate a struct that has type `ProverParam<Bls12_381>` with other related structs and functions since all of them are generic over `E`.
Another reason is we don't expect usage outside this (SRS) context, since caller already know type `E`, it's likely they don't have to specify any generic quanlifier at all. In contrast, if we specify `AffineRepr`, then caller would have to explicitly say `ProverParam<E::G1Affine>` each time.